unshaded_deploy.sh

```diff
@@ -3,8 +3,8 @@
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export GPG_KEY_ID="Snowflake Computing"
-export SONATYPE_USER="$sonatype_user"
-export SONATYPE_PWD="$sonatype_password"
+export LDAP_USER="$INTERNAL_NEXUS_USERNAME"
```

Comment:
Not this file, we should be updating unshaded_snapshot_deploy.sh?

Reply:
Sorry my bad. Updated the wrong file.Fixed the issue.